inc/saiswitch.h

```diff
@@ -3000,6 +3003,19 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_AVAILABLE_SYSTEM_VOQS,
 
+    /**
+     * @brief POE DEVICE LIST
+     *
+     * POE Device list.
+     * SAI_NULL_OBJECT_ID in the attribute value.
```

Comment:
Just repeqted brief description not needed

Reply:
Null object id has no meaning here what is that description? It does not explain anything

Reply:
I based the comments on SAI_SWITCH_ATTR_TAM_OBJECT_ID. I've removed the unneeded comments.